website/docs/r/cloud_project_database_mongodb_user.html.markdown

```diff
@@ -60,11 +60,8 @@ The following arguments are supported:
 
 * `service_name` - (Required, Forces new resource) The id of the public cloud project. If omitted,
   the `OVH_CLOUD_PROJECT_SERVICE` environment variable is used.
-
 * `cluster_id` - (Required, Forces new resource) Cluster ID.
-
-* `name` - (Required, Forces new resource) Name of the user.
-
+* `name` - (Required, Forces new resource) Name of the user. A user named "admin" is map with already created admin@admin user instead of create a new user.
```

Comment:
```suggestion
* `name` - (Required, Forces new resource) Name of the user. A user named "admin" is mapped with already created admin@admin user instead of creating a new user.
```